demo/kubernetes/rook.yml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: rest
+  labels:
+    app: rook
+    role: rest
+spec:
+  type: NodePort
+  externalIPs: [ "172.17.4.101" ]
+  ports:
+  - port: 8124
```

Comment:
Hardcoded IP addresses are doomed for failure - is there something better to put here?